usecase/usecase-wrt-android-tests/samples/Initialize/res/videos.html

```diff
@@ -0,0 +1,14 @@
+﻿<!DOCTYPE html> 
+<!-- Copyright © 2015 Michele Placentino,Adriano Perla. All Rights Reserved-->

```

Comment:
+<!-- Copyright © 2015 Michele Placentino,Adriano Perla. All Rights Reserved-->
@zqzhang Can we leverage this kind of sample as test cases directly in Crosswalk Test Suite?